deploy/dev/5_load_environment.sh

```diff
@@ -8,9 +8,20 @@ function main() {
 
   set_namespace "$APP_NAMESPACE_NAME"
 
-  configure_secret
+  if [ "${DEV}" = "false" ]; then
```

Comment:
when will this happen? isn't this script run only when `$DEV = true`?

Reply:
(it's inside `dev` folder)

Reply:
good point... It should have been `DEV_HELM`. Changing this now